utils/kata-manager.sh

```diff
@@ -669,16 +671,31 @@ handle_kata()
 	kata-runtime --version
 }
 
+containerd_version_number()
+{
+	local flavour="$1"
```

Comment:
Now that `flavour` has to be set, could you add a check on it. Something like:

```suggestion
	local flavour="${1:-}"
	[ -n "$flavour" ] || die "need containerd flavour"
```